docs/documentation/native/native2.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: Native Automation 2.0 (native2)
+---
+
+Patrol's **native2** introduces a unified approach to native mobile automation, providing cross-platform selectors and methods that work seamlessly across both iOS and Android platforms. This evolution from the original native automation API simplifies test writing while providing more powerful selector capabilities.
```

Comment:
It's not quite true - actually we separate selector classes to be platform-specific